exchange/c2cx/cli/c2cx.go

```diff
@@ -0,0 +1,424 @@
+// Package c2cx provides api methods methods for communication with c2cx exchange
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/shopspring/decimal"
+	"github.com/skycoin/exchange-api/exchange/c2cx"
+	"github.com/skycoin/skycoin/src/util/file"
```

Comment:
Use this library to get the home directory https://github.com/mitchellh/go-homedir
Don't use github.com/skycoin/skycoin